google/cloud/storage/benchmarks/CMakeLists.txt

```diff
@@ -45,6 +45,11 @@ target_link_libraries(
            google-cloud-cpp::storage_protos
            CURL::libcurl)
 google_cloud_cpp_add_common_options(storage_benchmarks)
+if (Protobuf_VERSION VERSION_LESS "23.x")
```

Comment:
Observation: Elsewhere you got rid of the need for ".x".

Reply:
Good point, done.